.github/workflows/bot.yml

```diff
@@ -511,6 +511,14 @@ jobs:
             flinkProfile: 'flink1.16'
             sparkProfile: 'spark3.3'
             sparkRuntime: 'spark3.3.1'
+          - scalaProfile: 'scala-2.12'
```

Comment:
We're going to remove the Spark 3.2 and 3.1 support.  We should have the following matrix:
```
          - scalaProfile: 'scala-2.13'
            flinkProfile: 'flink1.18'
            sparkProfile: 'spark3.5'
            sparkRuntime: 'spark3.5.1'
          - scalaProfile: 'scala-2.12'
            flinkProfile: 'flink1.17'
            sparkProfile: 'spark3.5'
            sparkRuntime: 'spark3.5.1'
          - scalaProfile: 'scala-2.12'
            flinkProfile: 'flink1.16'
            sparkProfile: 'spark3.4'
            sparkRuntime: 'spark3.4.3'
          - scalaProfile: 'scala-2.12'
            flinkProfile: 'flink1.15'
            sparkProfile: 'spark3.3'
            sparkRuntime: 'spark3.3.4'
          - scalaProfile: 'scala-2.12'
            flinkProfile: 'flink1.14'
            sparkProfile: 'spark3.3'
            sparkRuntime: 'spark3.3.1'
```

Reply:
I'll help generate and upload the docker images of bundle validation.

Reply:
Thanks for correcting me!  
I've fixed matrix and also fixed versions in ci_run.sh. And now I need your help with docker images.
`ERROR: docker.io/apachehudi/hudi-ci-bundle-validation-base:flink1180hive313spark351scala213: not found`

Reply:
@wombatu-kun Thanks for making the changes!  I'm going to upload the images today.

Reply:
I've put up the PR adding the new image build scripts: #11683

Reply:
All images are uploaded.  All bundle validations pass now in GH CI.